config/index.js

```diff
@@ -0,0 +1,179 @@
+var rp = require('request-promise');
+var fs = require('fs');
+var google = require('googleapis');
+
+var ENVIRONMENT = process.env.ENVIRONMENT || 'dev'
+var PROJECT_ID = `pi-ostelco-${ENVIRONMENT}`;
+var HOST = 'https://firebaseremoteconfig.googleapis.com';
+var PATH = '/v1/projects/' + PROJECT_ID + '/remoteConfig';
```

Comment:
Where is this path pointing? I'm unfamiliar with the /v1/projects .... , is that a coordinate inside firebase's URL space?

Reply:
PATH is used when calling the APIs, HOST + PATH = API and in some cases HOST + PATH + ':listVersions?pageSize=5'
There's nothing special and could've been done in many different ways.